src/preload.py

```diff
@@ -329,6 +330,16 @@ def resize_rootfs(additional_space):
     sfdisk(tmp.name, _in=new_layout)
     offsets_and_sizes = get_offsets_and_sizes(image, "s")
     copy = partial(ddd, _if=image, of=tmp.name, bs=SECTOR_SIZE, conv="notrunc")
+    # Copy data between the MBR and the first partition (some devices rely on
```

Comment:
I wonder if this would be clearer if reworded like "Copy across any data that's located between the MBR and the first partition ..." ?
I initially misinterpreted the line above as "Copy data from the MBR to the first partition ..." and had to `man dd` to work out what the arguments below are actually doing ;-)

Reply:
Sure, comment updated.